CMakeLists.txt

```diff
@@ -37,7 +37,12 @@ set_property(TARGET resilience PROPERTY CXX_STANDARD ${Kokkos_CXX_STANDARD})
 target_link_libraries(resilience PUBLIC Kokkos::kokkos)
 
 option(KR_ENABLE_VELOC "use VeloC backend for automatic checkpointing" ON)
-option(KR_ENABLE_STDFILE "use StdFile backend for automatic checkpointing" ON)
+option(KR_ENABLE_STDFILE "use StdFile backend for automatic checkpointing" OFF)
+
+option(KR_ENABLE_MAGISTRATE "use Magistrate for serializing and deserializing" OFF)
+option(KR_ENABLE_RESILIENT_EXEC "enable resilient execution spaces" OFF)
```

Comment:
ah just noticed this is wrong, see `KR_ENABLE_EXEC_SPACES` below. I think I messed this up in the rebase somewhere